lighthouse-core/gather/gather-runner.js

```diff
@@ -105,7 +105,9 @@ class GatherRunner {
   static async setupDriver(driver, options) {
     log.log('status', 'Initializing…');
     const resetStorage = !options.settings.disableStorageReset;
-    // Enable emulation based on settings
+    // In the devtools/extension case, we can't still be on the site while trying to clear state
+    // So we first navigate to a blank page, then apply our emulation & setup
+    await GatherRunner.loadBlank(driver);
```

Comment:
should this use the config for the first pass (for blank page and duration)?

Reply:
Nah, I think @paulirish was even borderline on the way to removing the `blankPage` and `blankDuration` params entirely from the passes anyway

Reply:
I'm a bit confused, aren't we still loading blank twice now? Seems like this line should go away and the comments moved into the `loadBlank` call in `run`

Reply:
sorry if I confused you with my "two blank calls" comment I meant for the pass and for before setup driver, not for setup driver + before setup driver :)